README.md

```diff
@@ -270,8 +272,7 @@ tool's purpose, then getting a high-level overview of the architecture
 and how it achieves the tool's purpose. With those concepts in place,
 concrete examples and exercises will help you understand how to use
 the tool. If you think that approach is bazonkers because you're a
-hands-on type purpose, you can skip to [01: schema, query,
-ent-db](#01-schema-query-ent-db).
+hands-on type purpose, you can skip to [01: ent-db](#01-ent-db).
```

Comment:
Is the phrasing `you're a hands-on type purpose` intentional? I believe `you're a hands-on type person` is what's intended.

Reply:
Fixed, but what is a person without purpose?